climada/hazard/trop_cyclone.py

```diff
@@ -627,16 +627,16 @@ def compute_windfields(track, centroids, model, metric="equirect"):
         return windfields, reachable_centr_idx
 
     # compute distances and vectors to all centroids
-    [d_centr], [v_centr] = u_coord.dist_approx(t_lat[None], t_lon[None],
-                                               track_centr[None, :, 0], track_centr[None, :, 1],
-                                               log=True, normalize=False, method=metric)
+    [d_centr], [v_centr_normed] = u_coord.dist_approx(
+        t_lat[None], t_lon[None], track_centr[None, :, 0], track_centr[None, :, 1],
+        log=True, normalize=False, method=metric)
```

Comment:
The array `v_centr` used to store the vectors from the track positions to the centroids. However, this array was never used, but only the normed version of it `v_centr_normed`. Only having a single array `v_centr_normed` instead reduces the number of critical arrays by 2 (since vectors have an x and a y component).